jsyntrax/src/test/java/org/atpfivt/jsyntrax/JSyntraxStyleFileTest.java

```diff
@@ -0,0 +1,155 @@
+package org.atpfivt.jsyntrax;
+
+import org.approvaltests.Approvals;
+import org.approvaltests.core.ApprovalFailureReporter;
+import org.approvaltests.core.Options;
+import org.atpfivt.jsyntrax.generators.SVGCanvas;
+import org.atpfivt.jsyntrax.generators.SVGCanvasBuilder;
+import org.atpfivt.jsyntrax.groovy_parser.Parser;
+import org.atpfivt.jsyntrax.styles.NodeStyle;
+import org.atpfivt.jsyntrax.styles.Style;
+import org.atpfivt.jsyntrax.styles.StyleConfig;
+import org.atpfivt.jsyntrax.units.Unit;
+import org.atpfivt.jsyntrax.units.tracks.stack.Stack;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import java.awt.*;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+public class JSyntraxStyleFileTest {
+
+    private final static Options options = new Options().forFile().withExtension(".svg");
+    private final SVGCanvasBuilder canvasBuilder;
+    private final Font testFont;
+
+    public JSyntraxStyleFileTest() {
+        testFont = getTestFont();
+
+        Style s = new Style(1, false);
+        s.title_font = testFont;
+
+        for (NodeStyle ns : s.nodeStyles) {
+            ns.font = testFont;
+        }
+        canvasBuilder = new SVGCanvasBuilder().withStyle(s);
+    }
+
+    private Font getTestFont() {
+        try {
+            return Font.createFont(Font.TRUETYPE_FONT,
+                    JSyntraxTest.class.getResourceAsStream("PTSans-Regular.ttf"))
+                    .deriveFont(12.f);
+        } catch (Exception e) {
+            throw new IllegalStateException(e);
+        }
+    }
+
+    @Test
+    void configParserTest() throws IOException {
+        Path stylePath = Paths.get("style.ini");
+        if (!Files.exists(stylePath)) {
+            Files.createFile(stylePath);
+        }
+        Assertions.assertTrue(Files.isRegularFile(stylePath));
+        StringBuilder config = new StringBuilder();
+        config.append("[style]\n")
+                .append("line_width = 50\n")
+                .append("v_sep = 42\n")
+                .append("text_color = (30, 40, 50)\n")
+                .append("shadow_fill = (35, 46, 57, 212)\n")
+                .append("[hex_bubble]\n")
+                .append("pattern = '^\\w'\n")
+                .append("shape = 'hex'\n")
+                .append("font = ('Sans', 14, 'bold')\n")
+                .append("fill = (255,15,3,129)");
+        Files.writeString(stylePath, config);
+        StyleConfig cfg = new StyleConfig(stylePath);
+
+        Assertions.assertEquals(cfg.line_width, 50);
```

Comment:
Too many asserts :-) 

First, we can decrease the number of asserts here.  Instead of three/four asserts for each color, you can use 

`Assertions.assertEquals(cfg.text_color, new Color(30,40,50));`
`Assertions.assertEquals(ns.fill, new Color(255, 15,3,129));`

Then, when we have multiple asserts, we should use 
```java
Assertions.assertAll(
    ()->assertEquals(...),
    ()->assertEquals(...),
    ...
);
```

I this case all the assertions will be checked even if some of them fail